docs/tools/draco.md

```diff
@@ -0,0 +1,84 @@
+# Draco: a Chemical Data Extractor
+
+[Draco] is a tool which aims to extract chemical data (Molecules, Reactions, procedures, etc.) from documents. In its first version, Draco focuses at extracting molecules (fragments and complete structures) from PDF documents
+
+## File Inputs
+
+### 1. PDF File
+
+Upload a PDF file containing chemical structures to extract to the Data Hub.
+
+## Parameters
+
+This section describes the parameters for a tool run, that are passed in the `start_run` function.
+
+### IUPAC name (OPTIONAL) - NOT YET AVAILABLE
+
+Use a model to predict the IUPAC name of the predicted structures
+
+## Outputs
+
+### Output Files
+
+Draco produces an .xlsx (Excel) file which contains all the extracted chemical strucutres for a given document. Each row contains the extracted image, the predicted image, the predicted SMILES, the confidence score and the confidence score associated to each token in the SMILES
+
+## Running Draco on Deep Origin
+
+To run Draco on Deep Origin, follow these steps:
+
+### 1. Create a database to store input and output files
+
+### 2. Start a tool run on Deep Origin
+
+For this tool run, we will use the following parameters:
+
+```python
+parameters = {
+    "input_file": 'your_pdf_file',
+}
+```
```

Comment:
why is this here? why not in the function argument?

Reply:
To be removed. This is not necessary as we specify input_file_column_name